composer.json

```diff
@@ -8,6 +8,7 @@
     "require": {
         "php": ">=7.4",
         "drupal/address": "^1.10",
+        "drupal/composite_reference": "^2.1",
```

Comment:
This belongs to the require-dev section, as it's required only in some submodules. We should then add in the README of the single submodule which parts are required.
See https://github.com/openeuropa/oe_content/tree/2.x/modules/oe_content_consultation#installation